src/components/search-results/index.js

```diff
@@ -7,13 +7,17 @@ import VisbilitySensor from 'react-visibility-sensor';
 import './style.css';
 
 const masonryOptions = {
-  transitionDuration: '0.4s',
+  transitionDuration: '0.2s',
   itemSelector: '.gridItem',
   fitWidth: true,
   gutter: 14 // horizontal spacing between tiles
 };
 
 class SearchResults extends Component {
+  constructor () {
+    super();
+    this.mapItems = this.mapItems.bind(this);
```

Comment:
The mapItems function needs to access `this.props` and with React Classes you need to explicitly bind `this` to the Class methods.